contracts/contracts/harvest/BaseHarvester.sol

```diff
@@ -221,15 +341,16 @@ abstract contract BaseHarvester is Governable {
     }
 
     /**
-     * @dev Swap all supported swap tokens for stablecoins via Uniswap.
+     * @dev Swap all supported reward tokens for
+     *      base token on the configured swap platforms
      */
     function swap() external onlyGovernor nonReentrant {
```

Comment:
I think we could get rid of all the governor only harvest and swap methods.

We've never used them since adding incentivized harvesting, and I can't see any reason we would need to use them in the future.

I think this would be:

- `harvest()`
- `swap()`
- `harvestAndSwap()`
- `harvest(address)`
- `swapRewardToken(address)`
- `_harvest() internal`
- `_swap(address)`

Reply:
Removed all of those now